main/main.cpp

```diff
@@ -656,6 +656,7 @@ void Main::print_help(const char *p_binary) {
 	print_help_option("--disable-crash-handler", "Disable crash handler when supported by the platform code.\n");
 	print_help_option("--fixed-fps <fps>", "Force a fixed number of frames per second. This setting disables real-time synchronization.\n");
 	print_help_option("--delta-smoothing <enable>", "Enable or disable frame delta smoothing [\"enable\", \"disable\"].\n");
+	print_help_option("--allow-gdriver-to", "Override the \"nvidia_disable_threaded_optimizations\" project setting.\n");
```

Comment:
That option's name looks strange to me. Is `gdriver` supposed to be "graphics driver"?

I would probably name it `--allow-threaded-optimization` instead.

Reply:
I did originally call this gpu in these PRs, but on reflection this isn't the gpu, this is the graphics driver, this feature is implemented on cpu, not gpu.

I'm just wary that we've had a number of cases of trying to "simplify" things for users (and using incorrect terms) that have backfired and caused misunderstandings.